src/main/java/com/nutomic/syncthingandroid/util/ConfigXml.java

```diff
@@ -152,6 +152,27 @@ private void updateIfNeeded() {
             }
         }
 
+        // Enforce TLS.
+        Element gui = (Element) mConfig.getDocumentElement()
+                .getElementsByTagName("gui").item(0);
+        boolean tls = Boolean.parseBoolean(gui.getAttribute("tls"));
+        if (!tls) {
+            Log.i(TAG, "Enforce TLS");
+            gui.setAttribute("tls", Boolean.toString(true));
+            changed = true;
+        }
+
+        // Update deprecated 8080 port to 8384
```

Comment:
I think Syncthing will never change the port because users are very much in control of the config.xml. For Syncthing-android this is definitely not the case and I think we fix a lot of problems (e.g. proxy servers) if we do not run on 8080 and upgrade existing clients with this (major) improvement.
Please reconsider..